.github/workflows/CI.yml

```diff
@@ -263,3 +263,30 @@ jobs:
         with:
           command: rustdoc
           args: --features full,ffi -- --cfg docsrs --cfg hyper_unstable_ffi -D broken-intra-doc-links
+
+  check-external-types:
+    name: Check exposed types
+    needs: [style, test]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

Comment:
We should pin to a specific combination of nightly version/cargo-check-external-types that's known to work. The newest nightly version might introduce a new version of rustdoc JSON output that's incompatible with the version expected by `cargo-check-external-type`.

It'd be nicer if we could install the toolchain version listed in their [toolchain file](https://github.com/awslabs/cargo-check-external-types/blob/main/rust-toolchain.toml), but that's probably not necessary to get this PR in.